### DISCUSSION TITLE
In the custom component, switch and label are used, and when 'justify-between' is used, the style does not take effect

### DISCUSSION THREAD

**Discussion by sshallow** [shadcn-ui/ui#3438] [General]
## Question

I want to divide the child components on both sides, so I use flex items-center justify-between, but it doesn't work, it's still on the left, why?

```
import { Label } from "@/components/ui/label"
import { Switch } from "@/components/ui/switch"

export function ATMSwitch({ id, label, onChange }) {
    const handleChange = (checked) => {
        if (onChange) {
            onChange(checked);
        }
    };

    return (
        <div className="flex items-center justify-between space-x-2">
            <Label htmlFor={id}>{label}</Label>
            <Switch id={id} onCheckedChange={handleChange} />
        </div>
    );
}

```

` <ATMSwitch id="filter-mode" label="Search Filter" onChange={handleSwitchChange} /> `
![image](https://github.com/radix-ui/primitives/assets/21257340/9fe4a5b6-9df2-46c1-9300-2bb576d29c77)

**Comment by DenserMeerkat** [comment]
Maybe ATMSwitch isn't taking the full width of its parent.
Try adding a `w-full` in ATMSwitch's div.
